src/RestEase/Implementation/PropertyGrouping.cs

```diff
@@ -61,6 +62,18 @@ public PropertyGrouping(IEnumerable<PropertyInfo> properties)
                     continue;
                 }
 
+                var requestPropertyAttribute = property.GetCustomAttribute<RequestPropertyAttribute>();
+                if (requestPropertyAttribute != null)
+                {
+                    AssertHasGetterAndSetter(property);
```

Comment:
We probably want to validate that 1) we don't have two properties with the same key, and 2) we don't have any params with the same key as either another param or a property. See `ValidatePathParams` for an example.